docker/jenkins/Dockerfile

```diff
@@ -12,4 +12,16 @@ RUN jenkins-plugin-cli --plugin-file /usr/share/jenkins/ref/plugins.yml
 # (Uncomment this line if you're using jenknis-casc-config.yml for auto-configuration)
 # ENV JAVA_OPTS -Djenkins.install.runSetupWizard=false
 
+# setup docker
+RUN apt-get update  \
```

Comment:
It is recommended to use `apt-get -y update` in scripts to disable interactive prompts when updating

Reply:
good idea